lagom/shopping-cart-scala/credentials.sbt

```diff
@@ -1,2 +1,11 @@
-credentials in ThisBuild += Credentials(Path.userHome / ".lightbend" / "commercial.credentials")
-resolvers in ThisBuild += "lightbend-commercial-maven" at "https://repo.lightbend.com/commercial-releases"
+resolvers in ThisBuild ++= {
+    val mvnResolver = sys.env.get("LIGHTBEND_COMMERCIAL_MVN").map { url =>
```

Comment:
When running locally, these need to be set from [Lightbend Platform Credentials](https://www.lightbend.com/account/lightbend-platform/credentials).